docs/csharp/whats-new/version-update-considerations.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Version and update considerations for C# developers
+description: Introducing new languages features in your library can impact the code that uses it.
+ms.date: 09/19/2018
+---
+
+# Version and update considerations for C# developers
+
+Compatibility is a very important goal as new features are added to the C# language. In almost all cases, you can use a new compiler version without breaking existing code.
```

Comment:
nit: start with "Backwards compatibility..."?

Reply:
I rewrote the 2nd sentence per @rpetrusha 's comment below. I think that addresses your comment as well. Thoughts?

Reply:
@BillWagner I like a new edition.

I don't even see the "Resolve conversation" button. I don't remember if it was there before your commit (that might be that the commit has resolved conversation automatically).

Reply:
Thanks @pkulikov I looks like it's a UI feature that's only shown to the PR author and committers. I've marked this as resolved.

Reply:
@BillWagner indeed. I see such a button in the PRs created by me.